lock-api-objects/src/main/java/com/palantir/lock/watch/IdentifiedVersion.java

```diff
@@ -16,14 +16,28 @@
 
 package com.palantir.lock.watch;
 
+import java.util.Comparator;
 import java.util.UUID;
 
 import org.immutables.value.Value;
 
+import com.palantir.logsafe.Preconditions;
+
 @Value.Immutable
 public interface IdentifiedVersion {
```

Comment:
As per James' comment, would like to rename this (in this PR or another).

Any thoughts to a good name?

Reply:
Let's worry about it for another PR.